sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -24,4 +44,1014 @@ public TextAnalyticsAsyncClient createTextAnalyticsAsyncClient() {
+
+            for (RecognizeEntitiesResult recognizeEntitiesResult : recognizeEntitiesResults) {
+                for (NamedEntity entity : recognizeEntitiesResult.getNamedEntities()) {
+                    System.out.printf(
+                        "Recognized Named Entity: %s, Type: %s, Score: %s.%n",
+                        entity.getText(),
+                        entity.getType(),
+                        entity.getScore());
+                }
+            }
+        });
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.recognizeEntities#List
+    }
+
+    /**
+     * Code snippet for {@link TextAnalyticsAsyncClient#recognizeEntitiesWithResponse(List, String)}
+     */
+    public void recognizeEntitiesWithResponseStringList() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.recognizeEntitiesWithResponse#List-String
+        List<String> textInputs1 = Arrays.asList(
+            "I had a wonderful trip to Seattle last week.", "I work at Microsoft.");
+
+        textAnalyticsAsyncClient.recognizeEntitiesWithResponse(textInputs1, "en").subscribe(response -> {
+            DocumentResultCollection<RecognizeEntitiesResult> recognizeEntitiesResults = response.getValue();
+            System.out.printf("Model version: %s%n", recognizeEntitiesResults.getModelVersion());
+
+            // Batch statistics
+            final TextDocumentBatchStatistics batchStatistics = recognizeEntitiesResults.getStatistics();
+            System.out.printf("Batch statistics, transaction count: %s, valid document count: %s.%n",
+                batchStatistics.getTransactionCount(),
+                batchStatistics.getValidDocumentCount());
+
+            for (RecognizeEntitiesResult recognizeEntitiesResult : recognizeEntitiesResults) {
+                for (NamedEntity entity : recognizeEntitiesResult.getNamedEntities()) {
+                    System.out.printf(
+                        "Recognized Named Entity: %s, Type: %s, Score: %s.%n",
+                        entity.getText(),
+                        entity.getType(),
+                        entity.getScore());
+                }
+            }
+        });
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.recognizeEntitiesWithResponse#List-String
+    }
+
+    /**
+     * Code snippet for {@link TextAnalyticsAsyncClient#recognizeBatchEntities(List)}
+     */
+    public void recognizeBatchEntities() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.recognizeBatchEntities#List
+        List<TextDocumentInput> textDocumentInputs = Arrays.asList(
+            new TextDocumentInput("0", "I had a wonderful trip to Seattle last week."),
+            new TextDocumentInput("1", "I work at Microsoft."));
+
+        textAnalyticsAsyncClient.recognizeBatchEntities(textDocumentInputs).subscribe(recognizeEntitiesResults -> {
+            System.out.printf("Model version: %s%n", recognizeEntitiesResults.getModelVersion());
```

Comment:
should we also remove the Model version log? Let us keep the codesnippet focus on the more valuable infor.

Reply:
removed